tests/app.spec.js

```diff
@@ -38,9 +44,21 @@ test('create a contact', async ({ page }) => {
     await page.fill('[aria-label="First Name"]', 'Test');
     await page.fill('[aria-label="Last Name"]', `Contact (${Date.now()})`);
 
+    // set a lookup value
+    const lookupFieldLabel = 'Account Name';
+    const lookupFieldTextValue = 'Northwind traders';
+    const clientLookup = page.locator(`[aria-label="${lookupFieldLabel}, Lookup"]`);
+
+    await clientLookup.hover();
+    await clientLookup.click();
+    await clientLookup.fill('');
+    await clientLookup.type(lookupFieldTextValue);
+
+    await page.click('[aria-label="Lookup results"]>li');
+
```

Comment:
Works on my machine! Thanks for adding this.

![chrome_vLpOui3J2j](https://user-images.githubusercontent.com/8723433/210179519-bb66ba35-ed4c-4d25-adfb-33834c50ac9e.gif)